packages/model-viewer/src/template.ts

```diff
@@ -333,7 +333,7 @@ canvas.show {
 
     <div class="slot progress-bar">
       <slot name="progress-bar">
-        <div id="default-progress-bar" aria-hidden="true">
+        <div id="default-progress-bar" part="default-progress-bar" aria-hidden="true">
```

Comment:
What kinds of styling will this part allow? Will we instead need a mask and a bar part?

Reply:
True. Probably better to remove this one for now.

Reply:
I think exposing parts on the two underlying divs would be better: default-progress-bar and default-progress-mask, say?